jobbergate-agent/Makefile

```diff
@@ -0,0 +1,40 @@
+SHELL:=/bin/bash
+ROOT_DIR:=$(shell dirname $(shell pwd)).PHONY: help
+
+.PHONY: lint
+lint: ## Run linter
+	tox -e lint
+
+.PHONY: format
+format: ## Run formatter
+	tox -e format
+
+.PHONY: get-lib
+get-lib: ## Get lib dependency
+	@charmcraft fetch-libs
+
+.PHONY: static
+static: get-lib ## Run static analysis
+	tox -e static
+
+.PHONY: qa
+qa: lint static ## Run all quality checks
+
+.PHONY: charm
+charm: ## Pack the charm
+	@git describe --tags --dirty --always > version
+	@charmcraft pack
+
+.PHONY: clean
+clean: ## Remove build dirs, temp files, and charms
+	@rm -rf lib/
+	@find . -type d -name ".ruff_cache" -exec rm -rf {} +
+	@find . -type d -name "venv" -exec rm -rf {} +
+	@find . -type d -name ".tox" -exec rm -rf {} +
+	@find . -name "*.charm" -delete
+	@find . -name "version" -delete
+	@find . -name "*config*.yaml" -delete
```

Comment:
isn't there any local files created by the `get-lib` target that should be defined here as well?

Reply:
It's the first line:
`@rm -rf lib/`

Reply:
oh great, I didn't notice, thank you